packages/api-rest/src/apis/common/handler.ts

```diff
@@ -26,6 +27,10 @@ type HandlerOptions = Omit<HttpRequest, 'body' | 'headers'> & {
 	body?: DocumentType | FormData;
 	headers?: Headers;
 	withCredentials?: boolean;
+	retryStrategy?: RetryStrategy;
+};
+const DEFAULT_RETRY_STRATEGY: RetryStrategy = {
+	strategy: 'jittered-exponential-backoff',
 };
```

Comment:
Very nit: `DEFAULT_RETRY_STRATEGY` can be just the string tag:
```suggestion
const DEFAULT_RETRY_STRATEGY = 'jittered-exponential-backoff' as const;
```